### PR TITLE
Add czmq

### PR DESCRIPTION
**Pull request by langmm**

### CODE REVIEW

**Comment by langmm** [comment]
It looks like there is not currently support for the MPL License. Will support be added in the future for MPL licenses or should this be classified as other?

**Comment by langmm** [comment]
FAQ says to ping after a week @conda-forge/staged-recipes. Is this correct?

File: recipes/czmq/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+requirements:
+  build:
+    - toolchain
+    - cmake
+    - autoconf  # [unix]
+    - automake  # [unix]
+    - libtool  # [unix]
+    - pkg-config  # [unix]
+    - libuuid  # [unix]
```
Comment (bgruening) [inline recipes/czmq/meta.yaml]: libuuid and pcre as well as zeromq should be in a host section, afaik

Comment (langmm) [reply recipes/czmq/meta.yaml]: Thank you for catching that. The most recent commit moves those packages and libuuid to the host section.

**Comment by bgruening** [comment]
Thanks @langmm!

**Pull request merged**